src/main/java/lotto/LottoMain.java

```diff
@@ -10,29 +9,28 @@
 
 public class LottoMain {
 
-  public static final String DELIMITER = ", ";
-
   public static void main(String[] args) {
     Money money = new Money(InputView.inputMoney());
-    int count = money.buyLotto();
-    Lottos lottos = LottoMachine.generateAuto(count);
-    lottos.print();
-    List<Integer> winnerNumber = getWinnerNumbers();
-    lottos.assignWinNumber(winnerNumber);
-    lottos.checkResult();
+    Lottos lottos = buyLottos(money);
+    ResultView.printLottos(lottos);
+
+    List<Integer> winnerNumber = InputView.inputWinnerNumber();
+    int bonusNumber = InputView.inputBonusNumber();
+    checkResult(lottos, winnerNumber, bonusNumber);
     ResultView.printResult(lottos);
     ResultView.printIncomePercent(lottos.calculateIncomePercent());
   }
 
-  private static List<Integer> getWinnerNumbers() {
-    String winnerNumberString = InputView.inputWinnerNumber();
-    String[] split = winnerNumberString.split(DELIMITER);
-    List<Integer> winnerNumber = new ArrayList<>();
-    for (String string : split) {
-      int number = Integer.parseInt(string);
-      winnerNumber.add(number);
-    }
-    return winnerNumber;
+  private static void checkResult(Lottos lottos, List<Integer> winnerNumber, int bonusNumber) {
+    lottos.checkResult(winnerNumber, bonusNumber);
   }
 
+  private static Lottos buyLottos(Money money) {
+    int count = money.getBuyableCount();
+    LottoMachine lottoMachine = new LottoMachine();
+    Lottos lottos = lottoMachine.generateAuto(count);
+    return lottos;
+  }
```

Comment:
1. LottoMachine 인스턴스를 매번 생성하지 않도록 상수로 선언하는건 어떨까요? 🙂 
2. lottos 지역변수없이 바로 리턴 해도 되겠네요!
```suggestion
  private static final LottoMachine LOTTO_MACHINE = new LottoMachine();
  
  private static Lottos buyLottos(Money money) {
    int count = money.getBuyableCount();
    return LOTTO_MACHINE.generateAuto(count);
  }
```